content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -73,24 +73,25 @@ Poseidon-Firmament scheduler does not provide support for high availability; its
 
 ## Feature comparison {#feature-comparison-matrix}
 
+{{< table caption="Features that are or are not supported by the Kubernetes default scheduler, compared to the Poseidon-Firmament scheduler." >}}
```

Comment:
Looking for a succinct table caption:
Feature comparison of Kubernetes and Poseidon-Firmament schedulers